packages/graphql_client/lib/models/nodes.dart

```diff
@@ -0,0 +1,1720 @@
+  BigInt? createdAt;
+  BigInt? updatedAt;
+  int? connectionPrice;
+  NodePower? power;
+  bool? dedicated;
+  BigInt? extraFee;
+
+  Node({
+    this.id,
+    this.gridVersion,
+    this.nodeID,
+    this.farmID,
+    this.twinID,
+    this.location,
+    this.country,
+    this.city,
+    this.publicConfig,
+    this.resourcesTotal,
+    this.uptime,
+    this.created,
+    this.farmingPolicyId,
+    this.interfaces,
+    this.certification,
+    this.secure,
+    this.virtualized,
+    this.serialNumber,
+    this.createdAt,
+    this.updatedAt,
+    this.connectionPrice,
+    this.power,
+    this.dedicated,
+    this.extraFee,
+  });
+
+  factory Node.fromJson(Map<String, dynamic> json) {
+    return Node(
+      id: json['id'] ?? "",
+      gridVersion: json['gridVersion'] ?? 0,
+      nodeID: json['nodeID'] ?? 0,
+      farmID: json['farmID'] ?? 0,
+      twinID: json['twinID'] ?? 0,
+      location: json['location'] != null
+          ? Location.fromJson(json['location'] as Map<String, dynamic>)
+          : null,
+      country: json['country'] ?? "",
+      city: json['city'] ?? "",
+      publicConfig: json['publicConfig'] != null
+          ? PublicConfig.fromJson(json['publicConfig'] as Map<String, dynamic>)
+          : null,
+      resourcesTotal: json['resourcesTotal'] != null
+          ? NodeResourcesTotal.fromJson(
+              json['resourcesTotal'] as Map<String, dynamic>)
+          : null,
+      uptime: BigInt.parse(json['uptime'] ?? '0'),
+      created: json['created'] ?? 0,
+      farmingPolicyId: json['farmingPolicyId'] ?? 0,
+      interfaces: json['interfaces'] != null
+          ? (json['interfaces'] as List)
+              .map((i) => Interfaces.fromJson(i as Map<String, dynamic>))
+              .toList()
+          : null,
+      certification: json['certification'] != null
+          ? certificationFromJson(json['certification'])
+          : null,
+      secure: json['secure'] ?? false,
+      virtualized: json['virtualized'] ?? false,
+      serialNumber: json['serialNumber'] ?? "",
+      createdAt: BigInt.parse(json['createdAt'] ?? '0'),
+      updatedAt: BigInt.parse(json['updatedAt'] ?? '0'),
+      connectionPrice: json['connectionPrice'] ?? 0,
+      power: json['power'] != null
+          ? NodePower.fromJson(json['power'] as Map<String, dynamic>)
+          : null,
+      dedicated: json['dedicated'] ?? false,
+      extraFee: BigInt.parse(json['extraFee'] ?? '0'),
+    );
+  }
+
+  @override
+  String toString() {
+    return generateToString(this);
+  }
+}
+
+class NodeEdgeInfo {
+  String? cursor;
+  Node? node;
+
+  NodeEdgeInfo({
+    this.cursor,
+    this.node,
+  });
+
+  factory NodeEdgeInfo.fromJson(Map<String, dynamic> json) {
+    return NodeEdgeInfo(
+      cursor: json['cursor'] ?? '',
+      node: json['node'] != null ? Node.fromJson(json['node']) : null,
+    );
+  }
+
+  @override
+  String toString() {
+    return generateToString(this);
+  }
+}
+
+class NodeConnectionsInfo {
```

Comment:
I got this error when trying to print an instance of <NodeConnectionsInfo> class.

```
Unhandled exception:
Reflecting on un-marked type "NodeConnectionsInfo"
#0      new _InstanceMirrorImpl (package:reflectable/src/reflectable_builder_based.dart:218:9)
#1      ReflectableImpl.reflect (package:reflectable/src/reflectable_builder_based.dart:2559:12)
#2      generateToString (package:graphql_client/src/query_builder.dart:97:45)
#3      NodeConnectionsInfo.toString (package:graphql_client/models/nodes.dart:1063:12)
#4      _StringBase._interpolate (dart:core-patch/string_patch.dart:866:19)
#5      main (file:///home/zainab/repos/tfgrid-sdk-dart/packages/graphql_client/bin/graphql_client.dart:10:21)
#6      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:297:19)
#7      _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)
```